### PR TITLE
Add Patient Filter for Review Missed

### LINKED ISSUE #6745: Patient Advanced Filter: filter by review missed
**Is your feature request related to a problem? Please describe.**
Currently there's no direct way to filter patients list by review missed.

**Describe the solution you'd like**
Support for filtering patients by review missed

### PR DESCRIPTION
**Pull request by jainvedant392**
## Proposed Changes

- Fixes #6745 
- Added filter field in PatientFilter.tsx for "Review Missed".
- Added filter badge for "Review Missed" in ManagePatients.tsx

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by Ashesh3 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -128,6 +128,7 @@
     "autoprefixer": "^10.4.14",
     "cypress": "^13.5.0",
     "cypress-localstorage-commands": "^2.2.3",
+    "dotenv": "^16.4.5",
```
Comment (Ashesh3) [inline package.json]: Why was this added? Context?

**Review by rithviknishad — APPROVED** [review]
Other than @Ashesh3 's comments, LGTM

File: package.json (hunk 1 above)
Comment (jainvedant392) [reply package.json]: Actually, I don't know why using env variables wasn't working in vite.config.mts, when I used "import 'dotenv/config'", it worked. That's the reason.

Comment (jainvedant392) [reply package.json]: that was onw of the reasons why I was unable to connect to the local care backend.

**Comment by jainvedant392** [comment]
Hey @rithviknishad, friendly reminder.

File: package.json (hunk 1 above)
Comment (rithviknishad) [reply package.json]: We are using [direnv](https://direnv.net/docs/installation.html) in this repository to load environment variables as configured [here](https://github.com/coronasafe/care_fe/blob/develop/.envrc)

Comment (rithviknishad) [reply package.json]: You could also connect to your local backend using: 
```
CARE_API=https://127.0.0.1:8000/ bun run dev
```

**Comment by rithviknishad** [comment]
@jainvedant392 ensure checks are passing. Cypress tests and Lint checks are failing.

**Comment by jainvedant392** [comment]
> @jainvedant392 ensure checks are passing. Cypress tests and Lint checks are failing.

Yeah I will check for that. Can you help me on how to ensure these checks are running successfully?

File: package.json (hunk 1 above)
Comment (jainvedant392) [reply package.json]: Ok, will try that.

Comment (jainvedant392) [reply package.json]: CARE_API=https://127.0.0.1:9000/ bun run dev
I added the above line in my .env, but unable to connect to the local backend which is running at http://localhost:9000/.

Comment (jainvedant392) [reply package.json]: What should I do now?

Comment (jainvedant392) [reply package.json]: Also, after removing dotenv, when I am using `process.env.CARE_API`, in `vite.config.mts`, its not working, but working when I am explicitly mentioning https://127.0.0.1:9000/.

Comment (rithviknishad) [reply package.json]: > CARE_API=[127.0.0.1:9000](https://127.0.0.1:9000/) bun run dev I added the above line in my .env, but unable to connect to the local backend which is running at [http://localhost:9000](http://localhost:9000/).

You shouldn't add that to the `.env`
Instead use that to run in the terminal directly.

Comment (jainvedant392) [reply package.json]: OK, if that's the thing, will try to do it.

Comment (jainvedant392) [reply package.json]: Hey, I have pushed some changes, could you please review them and suggest if any modification is required.

Comment (jainvedant392) [reply package.json]: Also all the tests are now passing successfully.

**Comment by Ashesh3** [comment]
Seems like all checks are passing - LGTM

File: vite.config.mts [hunk 2]
```diff
@@ -77,7 +79,7 @@ export default defineConfig({
     port: 4000,
     proxy: {
       "/api": {
-        target: process.env.CARE_API ?? "https://careapi.ohc.network",
+        target: local_care_api ?? "https://careapi.ohc.network",
```
Comment (rithviknishad) [inline vite.config.mts]: Why was this changed? The default dev. behaviour would become to use local backend with this change right? Could you revert it if it was unintentional?

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: vite.config.mts (hunk 2 above)
Comment (jainvedant392) [reply vite.config.mts]: Yeah I can revert it. The reason for this change was related to the previous problem which I mentioned, that the local backend wasn't connecting to the frontend through the env variable.

Comment (jainvedant392) [reply vite.config.mts]: I tried the above methods which you mentioned in the above thread.

Comment (jainvedant392) [reply vite.config.mts]: OK, I ran "CARE_API=http://127.0.0.1:9000/ bun run dev " and its working perfectly fine. I will push the requested changes.

Comment (jainvedant392) [reply vite.config.mts]: @rithviknishad

Comment (jainvedant392) [reply vite.config.mts]: Could you please mention the command and the way for resolving the failing cypress tests.

Comment (rithviknishad) [reply vite.config.mts]: See README: https://github.com/coronasafe/care_fe#-run-cypress-tests

```
bun run cypress:open
```

Comment (rithviknishad) [reply vite.config.mts]: Assuming dev. server is already running connected to your local BE

Comment (rithviknishad) [reply vite.config.mts]: Raise a seperate issue in case `CARE_API=... npm/yarn/pnpm dev` is not working.

**Comment by jainvedant392** [comment]
Hey @rithviknishad, I have written the test for this filter in the backend PR.

**Comment by nihal467** [comment]
@rithviknishad is it good for testing

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]
